src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyFPLaunchAuthenticator.ts

```diff
@@ -15,9 +15,11 @@ import {
   DescriptionElement,
   IdxStepTransformer,
   LaunchAuthenticatorButtonElement,
+  LinkElement,
   TitleElement,
 } from '../../../types';
 import { loc } from '../../../util';
+import { getCookieUserAuthenticated } from '../../../util/cookieUtils';
```

Comment:
I think you can just import from `../../../util` since there is a barrel file.

Reply:
done: https://github.com/okta/okta-signin-widget/commit/5abc64b4b5a05e743554b7552c095b3fd5078210